yarn-project/end-to-end/src/sample-dapp/contracts.mjs

```diff
@@ -1,11 +1,11 @@
-import { Contract } from '@aztec/aztec.js';
+import { AztecAddress, Contract } from '@aztec/aztec.js';
 import { TokenContractArtifact } from '@aztec/noir-contracts/artifacts';
 
 import { readFileSync } from 'fs';
 
 // docs:start:get-tokens
 export async function getToken(client) {
   const addresses = JSON.parse(readFileSync('addresses.json'));
-  return Contract.at(addresses.token, TokenContractArtifact, client);
+  return Contract.at(AztecAddress.fromString(addresses.token), TokenContractArtifact, client);
```

Comment:
Turns out we were passing a string when `Contract.at` wants an instance of `AztecAddress` and the only reason this worked was because the old `MemoryArchiverStore` was using `address.toString()` to find the contract. The new store actually really needs an address instance so this was crashing.

Might be worth going through all of the JS tests files and adding jsdoc typings so we at least spot issues in the editor.